dev/devicelab/manifest.yaml

```diff
@@ -279,23 +279,23 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["linux/android"]
 
-  gradle_plugin_test:
-    description: >
-      Verifies that the Flutter Gradle plugin supports standard and custom Android build types.
-    stage: devicelab
-    required_agent_capabilities: ["linux/android"]
-
-  plugin_test:
-    description: >
-      Checks that the project template works and supports plugins.
-    stage: devicelab
-    required_agent_capabilities: ["linux/android"]
-
-  module_test:
-    description: >
-      Checks that the module project template works and supports add2app on Android.
-    stage: devicelab
-    required_agent_capabilities: ["linux/android"]
+  # gradle_plugin_test:
```

Comment:
Lets leave a todo to remove the comments once we've verified this works well on cirrus